installer/buster/extensions/rdkafka.sh

```diff
@@ -8,20 +8,16 @@ function install_rdkafka()
         compile_rdkafka
     fi
 
-    docker-php-ext-enable rdkafka
+    if has_extension rdkafka; then
+        docker-php-ext-enable rdkafka
+    fi
```

Comment:
wouldn't this be confusing if someone tried to install it but it ended zero status with no output?

Reply:
True. Don't need to do it now php 8 has a build supported.